## This is the contents of a Tag
## Related pkg: eris
##
## Author: Ferdian Pratama

string name
string[] assoc_obj